riak_test/yz_siblings.erl

```diff
@@ -88,7 +137,8 @@ reconcile_sibs(Cluster, HP, Index, Bucket, EncKey) ->
 delete_key(Cluster, HP, Index, Bucket, EncKey) ->
     lager:info("Delete the key"),
     URL = bucket_url(HP, Bucket, EncKey),
-    {ok, "200", RH, _} = ibrowse:send_req(URL, [], get, [], []),
+    {ok, Status, RH, _} = ibrowse:send_req(URL, [], get, [], []),
+    ?assert(Status == "200" orelse Status == "300"),
```

Comment:
I am curious why checking for 300 is now required, but I am not completely up on the Riak HTTP API.  Will Riak send back a 300 after a partition heal?